core/task/sm/state_test.go

```diff
@@ -0,0 +1,88 @@
+package sm
```

Comment:
in general it's better to do `git mv <oldfile> <newfile>` instead of plain `mv`, since this works better with `git blame` and alike